CONTRIBUTING.md

```diff
@@ -387,23 +401,21 @@ Instance of freeCodeCamp](#maintaining-your-fork).
 1.  Perform the maintenance step of rebasing `staging`.
 2.  Ensure you are on the `staging` branch using `git status`:
 
-```bash
-$ git status
-On branch staging
-Your branch is up-to-date with 'origin/staging'.
-
-nothing to commit, working directory clean
-```
+        $ git status
+        On branch staging
+        Your branch is up-to-date with 'origin/staging'.
+        
+        nothing to commit, working directory clean
```

Comment:
btw, using indenting instead of fencing makes it so the numbered list does not restart at 1 after the code block -- Markdown is nuts